instrumentation/github.com/99designs/gqlgen/otelgqlgen/tags.go

```diff
@@ -0,0 +1,120 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package otelgqlgen
+
+import (
+	"fmt"
+
+	"github.com/vektah/gqlparser/v2/ast"
+	"github.com/vektah/gqlparser/v2/gqlerror"
+
+	"go.opentelemetry.io/otel/attribute"
+)
+
+const (
+	RequestVariablesPrefix        = "request.variables"
+	ResolverArgsPrefix            = "resolver.args"
+	ResolverErrorPrefix           = "resolver.error"
+	ServiceNameKey                = attribute.Key("service.name")
+	RequestQueryKey               = attribute.Key("request.query")
+	RequestComplexityLimitKey     = attribute.Key("request.complexityLimit")
+	RequestOperationComplexityKey = attribute.Key("request.operationComplexity")
+	ResolverPathKey               = attribute.Key("resolver.path")
+	ResolverObjectKey             = attribute.Key("resolver.object")
+	ResolverFieldKey              = attribute.Key("resolver.field")
+	ResolverAliasKey              = attribute.Key("resolver.alias")
+	ResolverHasErrorKey           = attribute.Key("resolver.hasError")
+	ResolverErrorCountKey         = attribute.Key("resolver.errorCount")
+)
```

Comment:
what is the point of exporting these consts?

Reply:
I believe they are simple abstractions to make it easier to pull certain attributes out of the GraphQL request object

Reply:
Correct, but it does not mean they have to be exported as the underlying values can change. Also if we would have a [semantic convention](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/trace/semantic_conventions) for GraphQL then it would be good to remove these consts as they would live in a package like https://pkg.go.dev/go.opentelemetry.io/otel/semconv/v1.4.0